packages/psss/src/components/Table/CellComponents/AlertCell.js

```diff
@@ -25,13 +25,11 @@ const StyledAlert = styled.div`
   }
 `;
 
-/*
- * Conditionally displays the value styled as an alert
- */
-export const AlertCell = props => {
-  const { displayValue } = props;
-  // this is just temporary logic until real data is in place
-  if (displayValue > 900) {
+export const AlertCell = ({ displayValue, columnKey, ...props }) => {
+  const key = `${columnKey} Threshold Crossed`;
```

Comment:
Again, that's a bit weird data key which I guess comes from the back-end...

I have a suspicion that the new PSSS server reports may use a field naming logic similar to the one used in tupaia.org. There, we need both the keys and the values that come from the back end to render reports which are pretty dynamic. An example:

```js
{ 
  data: [{ 'AFR Threshold Crossed': 3 }]
}
```

But in the case of PSSS, a data structure like the following may be more appropriate, if we use specific labels in the front-end:

```js
{ 
  data: [{ afrThresholdCrossed: 3 }]
}
```

It's probably not too bad either way, but just something to keep in mind.

Reply:
@rohan-bes - Just thought you might be interested in kostas's comment here